file_root/neutron/init.sls

```diff
@@ -16,13 +28,13 @@ neutron_conf_file:
       - ini: neutron_conf_file
   ini: 
     - options_present
-    - name: "{{ salt['pillar.get']('conf_files:neutron', default='/etc/neutron/neutron.conf') }}"
+    - name: "{{ salt['pillar.get']('conf_files:neutron') }}"
     - sections: 
         DEFAULT: 
           auth_strategy: keystone
           rpc_backend: neutron.openstack.common.rpc.impl_kombu
```

Comment:
rpc_backend should not be related in any way to the openstack version. You can use any of them, provided that it is configured ok. Please make sure there is no relation between the openstack version and the rpc backend.

Sent from mobile

On 31 ian. 2015, at 18:56, Chris Valean <notifications@github.com<mailto:notifications@github.com>> wrote:


In file_root/neutron/init.sls<https://github.com/cloudbase/salt-openstack/pull/3#discussion_r23890253>:

> @@ -16,13 +28,13 @@ neutron_conf_file:
>        - ini: neutron_conf_file
>    ini:
>      - options_present
> -    - name: "{{ salt['pillar.get']('conf_files:neutron', default='/etc/neutron/neutron.conf') }}"
> +    - name: "{{ salt['pillar.get']('conf_files:neutron') }}"
>      - sections:
>          DEFAULT:
>            auth_strategy: keystone
>            rpc_backend: neutron.openstack.common.rpc.impl_kombu


Pt Juno rpc_backend e rabbit, trebuie folosit variante de if pt juno/icehouse

—
Reply to this email directly or view it on GitHub<https://github.com/cloudbase/salt-openstack/pull/3/files#r23890253>.